app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -36,4 +37,24 @@ object FeatureFlags {
      * https://github.com/mozilla-mobile/fenix/issues/4431
      */
     const val mediaIntegration = true
+    val webChannelAuthentication = nightly or debug
+    val webChannelConfiguration = WebChannelConfigurations()
+
+
+    data class WebChannelConfigurations(
+        val redirectUrl: String =
+            if (webChannelAuthentication) {
+                "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
+            } else {
+                "https://accounts.firefox.com/oauth/success/${BackgroundServices.CLIENT_ID}"
+            },
+        val contentUrl: String =
+            if (webChannelAuthentication) {
+                //  TODO replace with release channel ^^
+                "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
```

Comment:
That's not right; the content URI doesn't change depending on if webchannel is on or off.

Reply:
Like, this _may_ be useful if we wanted to make it easier to switch over to dev/staging servers, but just use the `release` url for now (e.g., `SyncConfig.release`, not `SyncConfig(...)`

Reply:
I did it on that way because, I was taking a look at you previous commits where you change the content URI

Reply:
I think that was done just so that we can switch to a custom FxA server for testing. This PR should just use `release` regardless.

Reply:
Thanks, for the clarification. I addressed the commen